libminifi/src/RemoteProcessorGroupPort.cpp

```diff
@@ -20,6 +20,9 @@
 
 #include "../include/RemoteProcessorGroupPort.h"
 
+#include <curl/curl.h>
+#include <curl/curlbuild.h>
+#include <curl/easy.h>
```

Comment:
for now, i think we are tied couple curl with our code like others InvokeHttp, etc.